packages/salesforcedx-vscode-core/src/conflict/conflictNode.ts

```diff
@@ -43,7 +51,24 @@ export class ConflictNode extends vscode.TreeItem {
   }
 
   get tooltip() {
-    return this._conflict ? this._conflict.relPath : this.label;
+    if (this._conflict) {
+      let tooltipMessage: string = '';
+      if (this._conflict.remoteLastModifiedDate) {
+        tooltipMessage += nls.localize(
+          'conflict_detect_remote_last_modified_date',
+          `${new Date(this._conflict.remoteLastModifiedDate).toLocaleString()}`
```

Comment:
`this._conflict.remoteLastModifiedDate` and `this._conflict.remoteLastModifiedDate` are strings of format `2021-06-24T22:49:09.000Z` without time zone info. When we call `new Date(...)`, there is no time zone info stored in the Date object itself. When `toLocaleString` (and other functions of the Date object)  is used, the computer's local time zone is applied and displays the Date object in the local time zone (e.g. `6/24/2021, 10:49:09 PM`) ([reference](https://stackoverflow.com/questions/15141762/how-to-initialize-a-javascript-date-to-a-particular-time-zone )). We mentioned displaying time zone info in standup. Currently **not** displaying time zone info makes more sense to me because the org web page does not display time zone info. It might be confusing to users if we display locally. Open to suggestions!
<img width="348" alt="Screen Shot 2021-06-25 at 12 45 08 PM" src="https://user-images.githubusercontent.com/40456127/123477671-392c5100-d5b3-11eb-9dd7-8b64a8e5e431.png">

Reply:
The fact that it uses the computer's time zone is cool!

Reply:
FYI user can have a specific timezone set in Salesforce, possibly different than node runtime timezone, which can be queried from userInfo endpoint https://help.salesforce.com/articleView?id=sf.remoteaccess_using_userinfo_endpoint.htm&type=5. But you can imagine it won't be trivial setting and displaying the correct timezones and offsets. ;)

I'm fine with using `toLocaleString` etc without showing timezone as long as it provides enough info for user to understand the conflicts.

Reply:
We discussed this and decided to keep it as local timestamp on vscode side.